modules/ROOT/pages/dataweave-type-reuse-module-loader.adoc

```diff
@@ -0,0 +1,432 @@
+= Reusing XML Schema Module Loader
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can import XML schema files (`.xsd`) inside your DataWeave script as modules using the `xmlschema!` module loader. This loader allows you to use types declared on your schema inside DataWeave directly.
+DataWeave loads your XSD file and translates declarations inside your file into DataWeave type directives that can be accessed in the same way as types from any other DataWeave module. Use them to build new types, type check your variables, do pattern matching, or declare new functions that use types. There are no restrictions on how to use these types.
+
+The following example shows how to look for an XSD file in the path “org/weave/myfolder/User.xsd” being the first path the root of the classpath of your application:
+
+----
+import * from xmlschema!org::weave::myfolder::User
+----
+
+== Import Syntax
+
+In order to import the JSON schema types, as shown above, use the following directive:
+
+----
+import _typesToImport_ from jsonschema!_pathToXsdSchemaFile_
+----
+
+The following example shows how to import a type:
+
+----
+import * from jsonschema!example::schema::Person
+----
+
+Where:
```

Comment:
Maybe put the example above the syntax so that the "Where" follows the syntax bc `typeToImport` and `pathToXsdSchemaFile` are mentioned there.